policy/cis_command_output.cf

```diff
@@ -69,7 +80,12 @@ classes:
 
    "go_$(c)" expression => not(regcmp("$(regexp)","$(output)")),
              ifvarclass => strcmp("true","$(match)");
-   
+
+   # general debugging class specific for a bundle, should work with 3.6 & 3.7
+   "DEBUG_THIS_BUNDLE"
```

Comment:
Yes, I thought about this, too. Mine is an "and", yours obviously an "or". As it is style guide, we go for "or".
Will you fix it?